src/main/java/edu/ucla/library/services/metadata/CsvHeaders.java

```diff
@@ -43,6 +58,15 @@ public CsvHeaders(final String... aCsvHeaderRow) {
                 case "Format.extent":
                     myFormatExtentIndex = index;
                     break;
+                case "media.width":
```

Comment:
I noticed these values are already defined in MetadataSetter as `HEADER_WIDTH`, `HEADER_HEIGHT`, etc. Could we define them in a single place?